### PR TITLE
ADD TESTS TO SJ MANAGER

### PR DESCRIPTION
**Pull request by Reinkaos**
Acceptance Criteria:

The following files do not have specs:

- enrichment_jobs_controller.rb
- models/abstract_job.rb
- concerns/environment_helpers.rb
- models/enrichment_job
- parser_serializer.rb
- parser_version_serializer.rb
- snippet_version_serializer.rb
- source_serializer.rb

### CODE REVIEW

File: spec/models/abstract_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    it 'has a failed_records_count' do
+      expect(abstract_job.failed_records_count).to eq 0
+    end
+
+    it 'has a invalid_records_count' do
+      expect(abstract_job.invalid_records_count).to eq 0
+    end
+
+    it 'has a incremntal' do
```
Comment (theomarkkuspaul) [inline spec/models/abstract_job_spec.rb]: `incremental`

File: spec/serializers/snippet_version_serializer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+require 'spec_helper'
+
+RSpec.describe SnippetVersionSerializer do
+  let(:snippet) { create(:snippet) }
+  let(:version) { create(:version, versionable: snippet) }
+  let(:serializer) { described_class.new(version).as_json }
+
+  before(:each) do
+    allow(Snippet).to receive(:find).with(anything) { snippet }
```
Comment (theomarkkuspaul) [inline spec/serializers/snippet_version_serializer_spec.rb]: what is this `anything` argument you are passing here?

**Comment by theomarkkuspaul** [comment]
looks good man, thumbs up 👍 few comments left but they are more q+a

File: spec/serializers/snippet_version_serializer_spec.rb (hunk 2 above)
Comment (Reinkaos) [reply spec/serializers/snippet_version_serializer_spec.rb]: http://rspec.info/documentation/3.4/rspec-mocks/RSpec/Mocks/ArgumentMatchers.html#anything-instance_method :)

File: app/controllers/enrichment_jobs_controller.rb [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
+# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
 # http://digitalnz.org/supplejack
 
+# app/controllers/enrichment_jobs_controller.rb
 class EnrichmentJobsController < ApplicationController
-
   respond_to :js, :html
-
-  before_filter :set_worker_environment
+  before_action :set_worker_environment
```
[the remark this answers is not in the document]
Comment (Reinkaos) [reply app/controllers/enrichment_jobs_controller.rb]: Yeah bro :)

File: spec/helpers/environment_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe EnvironmentHelpers do
+  let(:env_vars) { APPLICATION_ENVIRONMENT_VARIABLES }
+  let(:params) { { environment: 'test' } }
+
+  class Dummy
```
[the remark this answers is not in the document]
Comment (Reinkaos) [reply spec/helpers/environment_helper_spec.rb]: Sure thing!

File: spec/models/abstract_job_spec.rb (hunk 1 above)
Comment (Reinkaos) [reply spec/models/abstract_job_spec.rb]: o0oo good catch

**Comment by ostigley** [comment]
🐔

**Pull request merged**